stagecoach/integration_tests/balance/app.py

```diff
@@ -1,77 +0,0 @@
-import asyncio
```

Comment:
If we need these I think we should import the logic from the balance monitors in stagecoach/monitors

Reply:
yeah in general i think once we're happy with how they're running we can refactor them to use the empiric library and move the logic out

Reply:
I just removed the files altogether and added a separate Dockerfile in the monitor folder that builds the Docker images we need to run these as tests in CI